### PR TITLE
 ✨ feature  - Remove some additional kcp references

### LINKED ISSUE #848: feature: finish adoption of definition of SyncTarget and Location
### Feature Description

Most of this work was done in #814 but there are comments in that PR and follow-ups that still need some attention.

### Proposed Solution

Follow up on all the comments on #814 and follow-ons.

### Want to contribute?

- [ ] I would like to work on this issue.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by fileppb**
## Summary
Remove some additional kcp references, as per Mike's comment:
 https://github.com/kubestellar/kubestellar/issues/848#issuecomment-1666147382
## Related issue(s)
#848
Fixes #848

### CODE REVIEW

**Comment by yana1205** [comment]
/lgtm

Thanks!
Just FYI: Tests have also passed
```
$ make test-syncer       
go test  `go list ./... | grep "/pkg/cliplugins\|/pkg/syncer"`
?       github.com/kubestellar/kubestellar/pkg/syncer   [no test files]
?       github.com/kubestellar/kubestellar/pkg/syncer/clientfactory     [no test files]
?       github.com/kubestellar/kubestellar/pkg/syncer/shared    [no test files]
?       github.com/kubestellar/kubestellar/pkg/syncer/syncers   [no test files]
ok      github.com/kubestellar/kubestellar/pkg/cliplugins/kubestellar/syncer-gen        0.827s
ok      github.com/kubestellar/kubestellar/pkg/syncer/controller        14.542s
```
```
$ make e2e-test-kubestellar-syncer
...
kcp is not ready. wait for 1s...
kcp is not ready. wait for 1s...
kcp is not ready. wait for 1s...
kcp is not ready. wait for 1s...
Starting test(s)
ok      github.com/kubestellar/kubestellar/test/e2e/kubestellar-syncer  156.216s
```

**Comment by fileppb** [comment]
@yana1205 Thanks, now I just need /approve

**Comment by fileppb** [comment]
@yana1205 thanks!

**Comment by fileppb** [comment]
@MikeSpreitzer would you /approve this?

**Review by clubanderson — COMMENTED** [review]
/approve

File: cmd/syncer/options/options.go [hunk 1]
```diff
@@ -53,7 +51,7 @@ func (options *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&options.ToKubeconfig, "to-kubeconfig", options.ToKubeconfig, "Kubeconfig file for -to cluster. If not set, the InCluster configuration will be used.")
 	fs.StringVar(&options.ToContext, "to-context", options.ToContext, "Context to use in the Kubeconfig file for -to cluster, instead of the current context.")
 	fs.StringVar(&options.SyncTargetName, "sync-target-name", options.SyncTargetName,
-		fmt.Sprintf("ID of the -to cluster. Resources with this ID set in the %q label will be synced.", workloadv1alpha1.ClusterResourceStateLabelPrefix+"<ClusterID>"))
+		fmt.Sprintf("ID of the -to cluster. Resources with this ID set in the %q label will be synced.", "<ClusterID>"))
```
Comment (MikeSpreitzer) [inline cmd/syncer/options/options.go:54]: I would have kept a prefix in this label's name (copying the definition of the constant symbol into the KubeStellar repo).

**Pull request merged**